inst/include/population_dynamics/population/population.hpp

```diff
@@ -377,7 +377,7 @@ struct Population : public fims_model_object::FIMSObject<Type> {
     Type phi0 = CalculateSBPR0();
     FIMS_LOG << "recruitment 2" << std::endl;
     FIMS_LOG << "phi0 = " << phi0 << std::endl;
-    FIMS_LOG << "spawning biomass = " << this->spawning_biomass[year]
+    FIMS_LOG << "spawning biomass = " << this->spawning_biomass[year - 1]
```

Comment:
This change seems to match what's noted in https://noaa-fims.github.io/collaborative_workflow/m1-model-specification.html#beverton-holt-recruitment-function, but I might change it to 
"spawning_biomass[year - 1] = " to make it extra clear what's being logged.

Reply:
Agree. The text has been changed to "spawning_biomass[year - 1] = ".